src/Pages/EmailPreferences/EmailPreferences.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { Container, Input, Button, Col } from 'reactstrap';
+
+import Header from '../../Components/Header/Header';
+import { setUserEmailPreference, getUserData } from '../../APIFunctions/User';
+
+export default function EmailPreferencesPage(props) {
+  const [isOptedIntoEmails, setIsOptedIntoEmails] = useState(false);
+  const [userEmail, setUserEmail] = useState('');
+  const [userFirstName, setUserFirstName] = useState('');
+  const [userLastName, setUserLastName] = useState('');
+  const [message, setMessage] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const urlParams = new URLSearchParams(window.location.search);
+      const userEmailParam = urlParams.get('user');
```

Comment:
if the userEmailParam is null, lets get out of this page, to navigate home we can do
```js
window.location = window.location.origin
```